Sofa/Component/Haptics/tests/scenes/ToolvsFloorCollision_test.scn

```diff
@@ -60,8 +60,8 @@
             <MeshOBJLoader filename="Demos/Dentistry/data/mesh/dental_instrument_centerline.obj"  name="loader"/>
             <MeshTopology src="@loader" name="InstrumentCollisionModel" />
             <MechanicalObject src="@loader" name="instrumentCollisionState"  ry="-180" rz="-90" dz="3.5" dx="-0.3" />
-            <LineCollisionModel contactStiffness="100"/>            
-            <PointCollisionModel contactStiffness="100"/>
+            <LineCollisionModel />
+            <PointCollisionModel />
```

Comment:
It does not change the test?

Reply:
it is the contrary ; there is now a warning if you set stiffness and you use FrictionConstraint as the response